games/2048.c/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem              1.0
 PortGroup               github 1.0
 PortGroup               makefile 1.0
 
-github.setup            mevdschee 2048.c 1.0.0 v
+github.setup            mevdschee 2048.c 1.0.1 v
 github.tarball_from     archive
 revision                1
```

Comment:
the `revision` field should be reset to `0` when updating the version

Reply:
Oops, forgotten the field was there. Changed now.